SnipperAPI/Controllers/SnippetsControllers.cs

```diff
@@ -1,15 +1,32 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using SnipperAPI.interfaces;
+
 
 namespace SnippetApi.Controllers
 {
     [ApiController]
     [Route("[controller]")]
     public class SnippetsController : ControllerBase
     {
-        private static readonly List<Snippet> Snippets = new List<Snippet>();
+        private static readonly List<EncryptedSnippet> Snippets = new List<EncryptedSnippet>();
+
+
+        private readonly IEncrypt _encryptionService;
+
+        public object EncryptionHelper { get; private set; }
+
+        public SnippetsController(IEncrypt encryptionService)
+        {
+            _encryptionService = encryptionService;
+        }
 
         // POST api/snippets
-        [HttpPost]
+        /*[HttpPost]
```

Comment:
Feel free to remove all this commented code as its pretty much redundant now you have the [HttpPost("snippets")] below